src/main/kotlin/domain/CalculationExpression.kt

```diff
@@ -0,0 +1,42 @@
+package domain
+
+import java.lang.IllegalArgumentException
+
+const val OPERATOR = 1
+class CalculationExpression(inputData: String?) {
```

Comment:
```suggestion
const val OPERATOR = 1

class CalculationExpression(inputData: String?) {
```

공백 라인을 활용한다면 가독성 좋게 문맥을 분리할수 있어요 😄